.github/workflows/generate-release-candidate.yml

```diff
@@ -0,0 +1,29 @@
+name: Generate Release Candidate
+
+on: 
```

Comment:
Add a schedule and comment it out, so we know this can be used for scheduled run as well.

Reply:
Sure.

Reply:
Added a commented schedule.